datafusion/physical-expr/src/functions.rs

```diff
@@ -964,47 +957,6 @@ fn func_order_in_one_dimension(
     }
 }
 
-/// This function specifies monotonicity behaviors for built-in scalar functions.
-/// The list can be extended, only mathematical and datetime functions are
-/// considered for the initial implementation of this feature.
-pub fn get_func_monotonicity(fun: &BuiltinScalarFunction) -> Option<FuncMonotonicity> {
```

Comment:
Can we please avoid a breaking api change by at least adding `pub use datafusion_expr::get_func_monotonicity` in this file? 

However, since we are going to move this code anyways, what  would you think about putting this code as a method on `BuiltinScalarFunction` itself, so that it was easier to find

Something like

```rust
impl BuiltinScalarFunction {
  pub fn monotonicity(&self) -> Option<FuncMonotonicity> {
  ...
   }
}
```

And leaving the `get_func_monotonicity` as a deprecated function, perhaps like

```rust
#[deprecated(message="use BuiltinScalarFunction::monotonicity")]
pub fn get_func_monotonicity(fun: &BuiltinScalarFunction) -> Option<FuncMonotonicity> {
  fun.monotonicity()
}
```

Reply:
Thank you, I also think deprecation is better